features/step_definitions/ui/view-and-card-brand-products.steps.js

```diff
@@ -0,0 +1,29 @@
+const { Given, When, Then } = require("@cucumber/cucumber");
+const { expect } = require("@playwright/test");
+
+Then("I verify that Brands are visible on left side bar", async function () {
+  const brandsSidebar = await this.page.locator('.brands_products');
+  await expect(brandsSidebar).toBeVisible();
+});
+
+When("I click on MADAME brand link", async function () {
+  const brandLink = await this.page.locator('a[href="/brand_products/Madame"]');
+  await brandLink.click();
+});
```

Comment:
seems like we can reuse this,  I this way this is universal for every brand.

seems like in a href, we have the brand name capitalize, so we can use something like this.

```
When( "I click on {string} brand link"), async function (name) {
  const name = utils.captialize(name);
  
   const brandLink = await this.page.locator('a[href=`/brand_products/${string}`]');
  await brandLink.click();
});
  

}

```

In new support file for ui.
```
function capitalize(str) {
  if (!str) return str; 
  return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
}
```